CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
 # Changelog
+## Unreleased
+
+### Added
+
+### Changed
+- [[#178](https://github.com/rust-vmm/vmm-sys-util/issues/178)]: Fixed the
```

Comment:
Let's be more explicit about what was happening: fixed a bug in `rand_bytes` that was triggering a panic when the number of bytes was not a multiple of 4.